fe/src/lib/components/svizzle/RangeSlider.svelte

```diff
@@ -164,7 +160,7 @@
 <div
 	{style}
 	class='RangeSlider'
-	on:touchstart={cancelPanelTouchStart}
+	on:touchstart|preventDefault
```

Comment:
Though it helps to get rid of the function and it uses a built-in Svelte feature, we should be aware that this also forwards the event to the public API for the component.